datahub-web-react/src/app/entity/shared/containers/profile/EntityProfile.tsx

```diff
@@ -345,6 +353,7 @@ export const EntityProfile = <T, U>({
                                                 refreshBrowser={refreshBrowser}
                                             />
                                             <EntityTabs
+                                                loading={loading}
                                                 tabs={[...tabsWithDefaults, ...autoRenderTabs]}
```

Comment:
The full tab set still contains all of them, whether they are enabled or not. 

What do you think about removing the changes to this file, and only changing the logic inside EntityTabs which changes the default selected tab? I do think that might be a bit simpler, with less duplication of logic.